content/en/blog/2025/go-opentelemetry-io-expired-certificate.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Retrospective for September 25th incident on go.opentelemetry.io
+linkTitle: Retrospective for September 25th incident on go.opentelemetry.io
```

Comment:
Don't include a `linkTitle` if it is the same as `title`. Otherwise, provide a shorter version such as:

```suggestion
linkTitle: Retrospective of go.opentelemetry.io incident
```